ignite/internal/plugin/consumer.go

```diff
@@ -0,0 +1,39 @@
+package plugininternal
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/ignite/cli/v28/ignite/pkg/errors"
+	"github.com/ignite/cli/v28/ignite/services/plugin"
+)
+
+// TODO move me to ignite org.
+const consumerPlugin = "github.com/tbruyelle/cli-plugin-consumer"
```

Comment:
https://github.com/ignite/cli/issues/3925

Reply:
Please note there's still some work to do on this PR. Recently a new `depinject.Inject()` call was added in `cmd/root.go` and this one is failing because of the absence of the staking keeper (which is replaced by the consumer keeper).

You can reproduce by scaffolding a consumer chain and run `ignite c serve -v`.

I'm still trying to find a solution, but a help would be apprecitated :)

Reply:
Sure, let me have a look.